delegate/shared/src/main/scala/zio/macros/delegate/annotations.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017-2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package zio.macros.delegate
+
+import scala.annotation.StaticAnnotation
+import scala.annotation.compileTimeOnly
+import scala.language.experimental.macros
+
+@compileTimeOnly("delegate annotation should have been removed.")
+class delegate(verbose: Boolean = false, forwardObjectMethods: Boolean = false, generateTraits: Boolean = true)
```

Comment:
I think for consitency we should either make this `Delegate` or lowercase the `Mockable` & `Accessable`.
Perhaps the latter would be better, as `zio-delegate` was around for some time already, so we'd like to keep the changes for the users to minimum, while probably the only user of Mockable & Accessable is me so far :joy:

Reply:
I don't think there are many users of delegate either :man_shrugging:
I believe annotations tend to be lowercase, so I would be in favor of making everything lower case